web-console/4.0-designs/user/user.md

```diff
@@ -0,0 +1,19 @@
+# User Menu
+
+## Copy Login Command
+![pic 0](img/copy-login-command-1.png)
+
+- Copy Login Command is the first option under the user menu in the masthead.
+
+![pic 1](img/copy-login-command.png)
```

Comment:
@beanh66 small nit, but can you change Logout to "Log out" ?  I know that Robb was putting that through this week (applies to all 3 images)

Reply:
> @beanh66 small nit, but can you change Logout to "Log out" ? I know that Robb was putting that through this week (applies to all 3 images)

FYI, that change is merged.

![projects openshift 2019-01-14 08-51-05](https://user-images.githubusercontent.com/1167259/51116596-9401d180-17d9-11e9-80ea-3512edd15be2.png)